src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +38,12 @@
      * "Can't get min value from list: method_input_list"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(stringNum -> Arrays.stream(stringNum.split(",")))
+                .mapToInt(Integer::parseInt)
+                .filter(num -> num % 2 == 0)
+                .min()
+                .orElseThrow(() -> new RuntimeException("Can't get min value from list"));
```

Comment:
```suggestion
                .orElseThrow(() -> new RuntimeException("Can't get min value from list: " + numbers));
```
let's make our message more informative

Reply:
Done